CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 /scripts/check_compliance.py				@nashif
 /scripts/what_changed.py                                @nashif
+/scripts/pylintrc                                       @ulfalizer
```

Comment:
Any other volunteer? Style issues can get easily get controversial...

Reply:
Could put you in if you want to.

Reply:
Can run `pylint --list-msgs` to see a list of messages btw. Most of them are enabled.

Types of stuff I skipped:

- Wrong hanging indent, because it'd require reformatting a ton of code, and it's too naggy. It 
  also flags deliberate alignment.

- Whitespace errors like `x = [ 1, 2, 3 ]` (instead of `x = [1, 2, 3]`). Too naggy, flags deliberate alignment. (I did a whitespace cleanup pass in a commit to fix all stuff like in the example though.)

- Stuff like avoiding `global` and other language features altogether. Too dogmatic.

- Naming stuff like never using super short variable names, and uppercasing global variables. Too dogmatic and naggy. Short names are fine if they're clear from context.

- Preferring

   ```python
   if foo:
       return bar
   return baz
   ```

   over

   ```python
   if foo:
       return bar
   else:
       return baz
   ```

   I like that one, but it might not always be a win, and it's too subjective.

Reply:
@marc-hb 
Added @mbolivar to CODEOWNERS.